packages/component-library/src/components/plmg-button/plmg-button.scss

```diff
@@ -278,6 +278,10 @@
   font-style: normal;
   font-weight: tokens.$plmg-font-weight-regular;
 
+  // Remove link styling
+  text-decoration: none;
+  width: fit-content;
```

Comment:
Why do you need to set the width to fit-content?

Reply:
Because it made the a-tag full width without

Reply:
Just realized it's not compatible with IE :confused: 
We did not really decide what was our position regarding supporting older browsers.